### PR TITLE
Draft - IBC Tooling

### PR DESCRIPTION
**Pull request by CitMC**
Documentation content update for the IBC module, section on tooling

### Change scope

The changes in this Pull Request include (please tick all that apply):

- [ ] Small language/grammar fixes
- [ ] Small content fixes 
- [x] Addition of new content
- [ ] Sample code/command updates
- [ ] Platform fixes
- [ ] Other

### R1: Internal review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [x] requested
  - [x] completed
- Technical platform review (remove if not required)
  - [ ] requested
  - [ ] completed

### R2: External review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [x] requested
  - [ ] completed

### R3: Internal QA review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R4: Final external QA review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### RC: Release Candidate

- [x] Ready to be merged

### CODE REVIEW

File: academy/ibc/ibc-tooling.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+In the meantime, introduce yourself in the channel ‘general-questions’.
+
+See you there!
+
+
+
+Noam and the Interchain Developer Academy Team
+
+More questions? Just hit reply and ask.
```
Comment (tmsdkeys) [inline academy/ibc/ibc-tooling.md:60]: Think someone accidentally pasted email content in there ;)

Comment (AndrewB9lab) [reply academy/ibc/ibc-tooling.md:60]: Yikes! Definitely me... thanks...

File: academy/ibc/ibc-tooling.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+* The IBC volume in (in USD)
+* A button for more `Details`
+
+If you click on the `Details`, you are directed to an overview with more in-depth information about the chain selected:
+
+![Detailed list for the Cosmos Hub](images/detailscosmoszone.png)
+
+In the overview, you can find a list of all the chains the selected chain is connected to. When you click a specific chain, you can see the channels between the selected chain and another chain:
+
+![Channels between the Cosmos Hub and Osmosis](images/cosmososmasischannels.png)
```
Comment (tmsdkeys) [inline academy/ibc/ibc-tooling.md:69]: This image or the other one on mintscan might be good opportunity to reiterate that there are canonical channels for ICS-20 and that all other channels will have been created accidentally by (unexperienced) relayers hence have practically no txs

**Review by tmsdkeys — APPROVED** [review]
Added a small improvement opportunity, but generally looks good

File: academy/ibc/ibc-tooling.md (hunk 2 above)
Comment (CitMC) [reply academy/ibc/ibc-tooling.md:69]: Will add the information in a highlight box in line 73. Should fit very well. Thank you for the input.

**Pull request merged**